### PR TITLE
Standardize Text throughout application

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #352 , #351 :*
https://github.com/bcgov/met-public/issues/352
https://github.com/bcgov/met-public/issues/351

-Add MetHeader 1,2,3,4 common component

-Add MetBody component

-Replace body and h1,2,3,4 typography variants throughout application for readability

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by jadmsaadaot — APPROVED** [review]
oh man major thanks for updating all the headers!

File: met-web/src/components/comments/admin/reviewListing/index.tsx [hunk 1]
```diff
@@ -129,9 +129,7 @@ const CommentListing = () => {
             <Grid item xs={0} md={4} lg={4}></Grid>
 
             <Grid item xs={12}>
-                <Typography variant="h4" sx={{ fontWeight: 'bold' }}>
-                    {`${comments[0]?.survey || ''} Comments`}
-                </Typography>
+                <MetHeader1 sx={{ fontWeight: 'bold' }}>{`${comments[0]?.survey || ''} Comments`}</MetHeader1>
```
Comment (tiagograf-AOT) [inline met-web/src/components/comments/admin/reviewListing/index.tsx:132]: I see a lot of entries where the fontWeight or other styles are getting changed anyway, this goes against the standardization, If the hearders are going to look the same in the whole app then these attributes should be incorporated in the component itself.

File: met-web/src/components/engagement/view/WhoIsListeningWidget.tsx [hunk 2]
```diff
@@ -13,9 +13,9 @@ const WhoIsListeningWidget = () => {
     return (
         <MetPaper elevation={1} sx={{ padding: '1em', minHeight: '12em' }}>
             <Grid item sx={{ pb: 2 }} justifyContent="flex-start" alignItems="center" xs={12}>
-                <Typography variant="h5" sx={{ fontWeight: 'bold' }}>
+                <MetHeader2 variant="h5" sx={{ fontWeight: 'bold' }}>
```
Comment (tiagograf-AOT) [inline met-web/src/components/engagement/view/WhoIsListeningWidget.tsx:16]: drop the variant

File: met-web/src/components/engagement/view/WhoIsListeningWidget.tsx [hunk 3]
```diff
@@ -35,14 +35,14 @@ const WhoIsListeningWidget = () => {
                     xs={8}
                 >
                     <Grid item xs={12}>
-                        <Typography
+                        <MetHeader3
```
Comment (tiagograf-AOT) [inline met-web/src/components/engagement/view/WhoIsListeningWidget.tsx:38]: drop the variant

File: met-web/src/components/comments/admin/reviewListing/index.tsx (hunk 1 above)
Comment (jadmsaadaot) [reply met-web/src/components/comments/admin/reviewListing/index.tsx]: Good point, but I think the BC standardization isn't setting the bold option here for all h1s. Will MetHeader1 always be bold? Maybe let's confirm with MA @djnunez-aot

Comment (djnunez-aot) [reply met-web/src/components/comments/admin/reviewListing/index.tsx]: Font weight 500 is the header standard with Marie Anik's design

File: met-web/src/components/engagement/view/WhoIsListeningWidget.tsx (hunk 2 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/view/WhoIsListeningWidget.tsx]: dropped

File: met-web/src/components/common/index.tsx [hunk 4]
```diff
@@ -175,3 +173,43 @@ export const modalStyle = {
...
+
+export const ModalSubtitle = ({
+    children,
+    ...rest
+}: {
+    children: JSX.Element[] | JSX.Element | string;
+    [prop: string]: unknown;
+}) => {
+    return <Typography variant={'subtitle1'}>{children}</Typography>;
+};
```
Comment (jadmsaadaot) [inline met-web/src/components/common/index.tsx:215]: =
6d406103-c094-4950-99d8-058c0fa88636
Have all of these been added (including paragraph, small text, link, maybe the link is already done in theme)

Comment (djnunez-aot) [reply met-web/src/components/common/index.tsx:215]: yes Link is already configured and met label as well that's why I only focused on header and body

**Comment by tiagograf-AOT** [comment]
@djnunez-aot just some lint errors!

File: met-web/src/components/common/index.tsx (hunk 4 above)
Comment (jadmsaadaot) [reply met-web/src/components/common/index.tsx:215]: paragraph and small text?

**Review by tiagograf-AOT — APPROVED** [review]

**Pull request merged**